pkg/reconciler/openshift/tektonaddon/communityClusterTasks.go

```diff
@@ -0,0 +1,140 @@
+package tektonaddon
+
+import (
+	"context"
+	"fmt"
+	mf "github.com/manifestival/manifestival"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/operator/pkg/reconciler/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/logging"
+	"strings"
+	"time"
+)
+
+var communityClusterTaskLS = metav1.LabelSelector{
+	MatchLabels: map[string]string{
+		v1alpha1.InstallerSetType: CommunityClusterTaskInstallerSet,
+	},
+}
+
+// After only holds value until the pod is not restarted.
+// ones the pod restart resets the value back to ""
+// this is useful if the pod is not restarted and community task urls are not reachable
+// we see comparable less forbidden events
+var (
+	TimeFormat = "2006-01-02 15:04:05"
+	After      = ""
```

Comment:
`After` please rename this to make the purpose of this variable explicit.

note:  Idiomatic Go values comprehensibility over conciseness